components/back-compatile-on-input/index.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Typo on file name: `back-compatile-on-input` -> `back-compatible-on-input`

Aside: Not sure I love the name, mostly because it's an abbreviation of "backwards", though also thinking there might be some alternative like `NormalizedOnInput` or `ContentEditable`